docs/aurora/known-issues.md

```diff
@@ -1,12 +1,92 @@
+
+#### DAOS (object store)
+
+Performance of DAOS has been impressive, but we continue to experience crashes with large jobs, including loss of data. Projects may use it, but should not consider it stable or safe for long-term storage.
+
+#### Grand/Eagle
+
+These won’t be mounted on Aurora initially, but they might be mounted around May 2025, depending on feasibility. Similarly, Flare will not initially be mounted on Polaris. DTNs and [Globus](../data-management/data-transfer/using-globus.md) are the best means to transfer data between Polaris and Aurora.
+
+### Checkpointing 
+
+Checkpointing is absolutely essential. The mean time between application interrupts caused by system instability may be as short as an hour for larger jobs. The frequency of checkpointing is something that needs to be decided for each individual application based on scale of runs.
+
+* If checkpointing has minimal overhead, consider checkpointing once every 15 minutes.
+
+* If checkpointing has substantial overhead, then consider checkpointing every 30-60 minutes
+
+* It may be the case that the highest throughput initially will be with creating job dependency chains where scripts are able to 1) automatically restart from the latest available checkpoint file and 2) confirm that the prior run generated reasonable/correct results
+
+### Troubleshooting Common Issues
+
+As always, INCITE and ALCC projects should Report all issues to Catalyst point of contact.
+
+#### Ping Failures
+
+Network instabilities may lead to some nodes or MPI ranks losing communication with others. If your job output shows an error message like these:
+
+```
+ping failed on x4707c6s4b0n0: Couldn't forward RPC ping(24c93b8c-3434-4fb5-a8f0-53cff4cbbe42) to child x4707c7s6b0n0.hostmgmt2707.cm.aurora.alcf.anl.gov: Resource temporarily unavailable
+```
+
+```
+ping RPC timeout from x4212c7s0b0n0.hostmgmt2212.cm.aurora.alcf.anl.gov after 120s
+```
+then most likely your application will crash or hang. If you see these, the best action is to kill the job and re-run it (from the last checkpoint, if there has been one).
+
+<!-- #### PBS failures -->
+
+<!-- Need to fill this in with examples and advice to set retry_count to something sane -->
+
+#### Hangs
+
+There are multiple failure modes that can lead to jobs hanging. For known hardware or low-level software issues such as ping failures as discussed above, just restart the job. 
+
+To avoid a hung job running out all the requested wallclock time on all its nodes, we suggest devising ways to monitor job progress. For example, if your application regularly writes small output to logfile, then you could launch a “watcher” script look for that expected output and collect a stack trace and kill the job if it's been too long.
```

Comment:
```suggestion
To avoid a hung job running out all the requested wallclock time on all its nodes, we suggest devising ways to monitor job progress. For example, if your application regularly writes small output to logfile, then you could launch a “watcher” script that looks for that expected output and collect a stack trace and kill the job if it's been too long since progress was made. Please engage your Catalyst POC if you are interested in evaluating this for your application.
```